Userland/Libraries/LibGL/Tex/TextureUnit.h

```diff
@@ -35,6 +36,8 @@ class TextureUnit {
     bool texture_cube_map_enabled() const { return m_texture_cube_map_enabled; };
     void set_texture_cube_map_enabled(bool texture_cube_map_enabled) { m_texture_cube_map_enabled = texture_cube_map_enabled; }
 
+    IntrusiveListNode<TextureUnit> m_bound_node;
```

Comment:
In the kernel we usually expose the list as a new nicely named subtype. Should we follow suite here?

Reply:
Originally I thought another layer of indirection worked confusingly, but I think defining a subtype is nicer. Added!